source-front/source_front/__init__.py

```diff
@@ -1,8 +1,63 @@
-#
-# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
-#
+from logging import Logger
+from typing import Callable, Awaitable
 
+from estuary_cdk.flow import (
+    ConnectorSpec,
+)
+from estuary_cdk.capture import (
+    BaseCaptureConnector,
+    Request,
+    Task,
+    common,
+    request,
+    response,
+)
+from estuary_cdk.http import HTTPMixin
 
-from .source import SourceFront
+from .resources import all_resources, validate_credentials
+from .models import (
+    ConnectorState,
+    EndpointConfig,
+    ResourceConfig,
+)
 
-__all__ = ["SourceFront"]
+
+class Connector(
+    BaseCaptureConnector[EndpointConfig, ResourceConfig, ConnectorState],
+    HTTPMixin,
+):
+    def request_class(self):
+        return Request[EndpointConfig, ResourceConfig, ConnectorState]
+
+    async def spec(self, _: request.Spec, logger: Logger) -> ConnectorSpec:
+        return ConnectorSpec(
+            configSchema=EndpointConfig.model_json_schema(),
+            documentationUrl="https://go.estuary.dev/source-pendo",
```

Comment:
Documentation URL needs updated

Reply:
:facepalm: Thanks for catching the copy-paste remnants I missed (again)!